### PR TITLE
First cut of defining separate SC / SB models with changes reflected …

### PR DESCRIPTION
**Pull request by vaikas-google**
…in the controller

Signed-off-by: vaikas-google <vaikas@google.com>

### CODE REVIEW

File: k8s/service_controller/server/controller.go [hunk 1]
```diff
@@ -8,8 +8,9 @@ import (
 	"strings"
 	"time"
 
-	"github.com/cncf/servicebroker/k8s/service_controller/model"
 	"github.com/cncf/servicebroker/k8s/service_controller/utils"
+	sb_model "github.com/cncf/servicebroker/model/service_broker"
```
Comment (bmelville) [inline k8s/service_controller/server/controller.go]: i think typically in go it's just sbmodel and scmodel

File: model/service_broker/service_broker.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package model
```
Comment (bmelville) [inline model/service_broker/service_broker.go]: This is a SC model I think. Service broker doesn't actually know about itself :)

**Comment by bmelville** [comment]
I'm assuming that this is just separating them out without modifying the models to align with the actual SC model? If so, just the one service broker comment, otherwise looks good.

File: k8s/service_controller/server/controller.go (hunk 1 above)
Comment (vaikas-google) [inline k8s/service_controller/server/controller.go]: done.

**Comment by bmelville** [comment]
LGTM

**Comment by vaikas-google** [comment]
Thanks for the review, PTAL.

**Comment by duglin** [comment]
let's not merge any more PRs until we get travis running.  Will also review this later today on the plane.

**Comment by vaikas-google** [comment]
Who's in charge of setting up the travis and what's the expected ETA?

**Comment by duglin** [comment]
I'm working with @caniszczyk to try to get it fixed.

**Comment by duglin** [comment]
@caniszczyk added travis back so we should be good now

**Comment by bmelville** [comment]
LGTM

**Pull request merged**